astro/src/content/docs/lifecycle/migrate-users/bulk/stytch.mdx

```diff
@@ -0,0 +1,381 @@
+
+<Aside type="note">
+The linked repository has multiple plugin projects. You need to build and install only the hashing project.
+</Aside>
+
+Download and unzip the repository, or use Git in a terminal with the code below.
+```sh
+git clone https://github.com/fusionauth/fusionauth-contrib.git
+```
+
+Open the file `fusionauth-contrib/Password Hashing Plugins/src/main/java/com/mycompany/fusionauth/plugins/ExampleStytchScryptPasswordEncryptor.java`. The content of the file is shown below.
+
+<RemoteCode
+    url={frontmatter.javaCodeRoot + "/Password Hashing Plugins/src/main/java/com/mycompany/fusionauth/plugins/ExampleStytchScryptPasswordEncryptor.java"}
+    lang="java"
+/>
+
+This program takes a password and salt and returns a hash. It accepts and returns the `+` form of Base64, but changes it to `-` to work with Java internally. The `factor` parameter is ignored. Scrypt instead has multiple parameters set at the top of the file. If any of these differ from the ones you received in your encrypted CSV file header, you need to change them in the Java file.
+
+There are two other files linked to this scrypt plugin that you shouldn't need to alter:
+- `fusionauth-contrib/Password Hashing Plugins/src/main/java/com/mycompany/fusionauth/plugins/ExampleStytchScryptPasswordEncryptorTest.java`, which holds unit tests for the hasher.
+- `fusionauth-contrib/Password Hashing Plugins/src/main/java/com/mycompany/fusionauth/plugins/guice/MyExampleFusionAuthPluginModule.java`, which makes the hasher file known to FusionAuth.
+
+If you make another hash plugin, you will need to make your own test and add your plugin to the Guice file.
+
+Build the Java plugin and add it to FusionAuth:
+- Open a terminal and run the code below to build a Docker container for Java, with the repository shared as a volume into the container.
+  ```sh
+  cd fusionauth-contrib/.devcontainer
+  docker build -t javaimage .
+  cd ..
+  docker run -it --name javabox -v .:/workspace javaimage
+  ```
+- In the Docker container terminal that is now running, run the code below.
+  ```sh
+  cd "/workspace/Password Hashing Plugins"
+  mvn clean install
+  exit
+  ```
+- If you have Java installed locally, you can run the Maven command without Docker.
+
+All tests should pass, and the plugin file should be available in `Password Hashing Plugins/target/fusionauth-example-password-encryptor-0.1.0.jar`.
+
+### Set Up FusionAuth And Deploy The Plugin
+
+Now copy `fusionauth-example-password-encryptor-0.1.0.jar` to your FusionAuth `plugins` directory and restart FusionAuth.
+
+If you are not already running FusionAuth or want to test this process on another instance, you can start FusionAuth in Docker.
+- Open a new terminal in the `fusionauth-import-scripts` directory and run the code below.
+  ```sh
+  cd stytch/fusionAuthDockerFiles
+  docker compose up
```

Comment:
Stuck here.  Get Error Message: 

fa     | 2024-03-07 10:36:42.339 PM INFO  com.inversoft.maintenance.search.ElasticsearchSilentModeWorkflowTask - Attempt 100 of 100. Waiting for search to become available [SERVER_DOWN]. Error [http://search:9200].